### PR TITLE
Add documentation for how to make a release

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

File: docs/how_to_release.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Only [maintainers](https://github.com/orgs/OverlayPlugin/teams/maintainers/members) can currently publish a release.
+If you'd like to be a maintainer, please contact anybody on that list.
+
+## Steps
+
+### Update the AssemblyInfo versions
+
+Run `python tools/set_version.py 0.19.0`
+where `0.19.0` is the version you want to update.
+This will update all of the `AssemblyInfo.cs` files to the new version number.
```
Comment (panicstevenson) [inline docs/how_to_release.md:17]:
```md
Optional: Run `python tools/validate_version.py` after to ensure all versions are in sync.
```

Comment (quisquous) [reply docs/how_to_release.md:17]: Done.

**Review by panicstevenson — APPROVED** [review]

File: docs/how_to_release.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# OverlayPlugin Release Process
+
+This doc describes how to make a release of OverlayPlugin.
+
+## Overview
+
+Anybody can (and should) propose a release by sending a PR if they think a release is needed.
+Only [maintainers](https://github.com/orgs/OverlayPlugin/teams/maintainers/members) can currently publish a release.
```
Comment (Makar8000) [inline docs/how_to_release.md:8]: Might want to check the URL and/or the permissions of the maintainer list (if that's even a thing that can be set). I am getting a 404 error when accessing https://github.com/orgs/OverlayPlugin/teams/maintainers/members
...or maybe the team just hasn't been setup yet?

Comment (panicstevenson) [reply docs/how_to_release.md:8]: Out of curiosity, does it show anything now? I put myself as a public member, whereas before everyone was private

Comment (Makar8000) [reply docs/how_to_release.md:8]: Still 404

Comment (quisquous) [reply docs/how_to_release.md:8]: Yeah, I don't see any visibility settings.  It says the team is public.  It doesn't let me look at it without being logged in, but I don't have another account to use that's not me to see what that looks like.  I'll remove the link.

**Review by panicstevenson — APPROVED** [review]

**Pull request merged**